packages/server/database/migrations/20230206131155-updateOrgIdOnSAML.ts

```diff
@@ -0,0 +1,16 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
```

Comment:
As of [7720](https://github.com/ParabolInc/parabol/pull/7720) we now store all migrations in  `sevrer/postgres/migrations`. For this reason the build fails. Can you move the migration there?

Reply:
You would need to run `yarn pg:migrate create updateOrgIdOnSAML.ts`.
Then copy your migration there
Update migration code by adding `connectRethinkDB` see example here: https://github.com/ParabolInc/parabol/blob/eeb6d0282a8e92688a4fab92a2338d76b5d3c6f3/packages/server/postgres/migrations/1674074684762_deleteEmptyGroups.ts